helpers/address.go

```diff
@@ -11,6 +11,12 @@ import (
 	"github.com/decred/dcrd/hdkeychain/v2"
 )
 
+const (
+	// ExternalBranch is a helper value that needs to
+	// match dcrwallet's udb.ExternalBranch
+	ExternalBranch = 0
```

Comment:
Could you also keep the type, uint32?
https://github.com/decred/dcrwallet/blob/d273b5687adfac4e74ba3fa21370da31158b1580/wallet/udb/addressmanager.go#L76

Reply:
I'm curious why this should matter?

Reply:
Because that's what it is. And the comment above says it needs to match...